pkg/template/template.go

```diff
@@ -130,6 +131,11 @@ func Str2html(raw string) template.HTML {
 	return template.HTML(markup.Sanitize(raw))
 }
 
+// Simple filter, converts newline symbols to <br>
+func NewLine2br(raw string) template.HTML {
+	return template.HTML(strings.Replace(template.HTMLEscapeString(raw), "\n", "<br>", -1))
```

Comment:
I think it would be more clear that just do `strings.Replace(raw, "\n", "<br>", -1)`

Reply:
Done